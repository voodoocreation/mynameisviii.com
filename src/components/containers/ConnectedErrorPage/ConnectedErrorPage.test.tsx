import { render } from "enzyme";
import * as React from "react";
import { Provider } from "react-redux";

import createStore from "../../../store/root.store";
import ConnectedErrorPage from "./ConnectedErrorPage";

const setup = (fn: any) => {
  const props = {
    error: {
      message: "Not found",
      status: 404
    }
  };
  const store = createStore();

  return {
    actual: fn(
      <Provider store={store}>
        <ConnectedErrorPage {...props} />
      </Provider>
    ),
    props
  };
};

describe("<ConnectedErrorPage />", () => {
  it("renders correctly", () => {
    const { actual } = setup(render);
    expect(actual).toMatchSnapshot();
  });
});
