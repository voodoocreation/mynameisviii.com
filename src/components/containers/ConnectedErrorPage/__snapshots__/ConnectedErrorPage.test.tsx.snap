// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConnectedErrorPage /> renders correctly 1`] = `
Array [
  <header
    class="PageHeader"
  >
    <h1>
      An error has occurred
    </h1>
  </header>,
  <p>
    An error has occurred, please try again.
  </p>,
]
`;
